prediction-polls/android/app/build.gradle.kts

```diff
@@ -66,4 +70,22 @@ dependencies {
     androidTestImplementation("androidx.compose.ui:ui-test-junit4")
     debugImplementation("androidx.compose.ui:ui-tooling")
     debugImplementation("androidx.compose.ui:ui-test-manifest")
+
+    // Dagger Hilt
+    implementation("com.google.dagger:hilt-android:2.48.1")
+    kapt("com.google.dagger:hilt-compiler:2.48.1")
```

Comment:
why did you prefer kapt instead of ksp? is there any specific reason?

Reply:
Nope no specific reasons if you want I can change to ksp.

Reply:
ksp is up to two times more performant I think it would be better

Reply:
kk I wil replace it.